source/logrepl/combined_test.go

```diff
@@ -0,0 +1,272 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logrepl
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/conduitio/conduit-connector-postgres/source/position"
+	"github.com/conduitio/conduit-connector-postgres/test"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/google/go-cmp/cmp"
+	"github.com/matryer/is"
+)
+
+func TestConfig_Validate(t *testing.T) {
+	is := is.New(t)
+
+	err := Config{
+		Tables: []string{
+			"t1", "t2", "t3", "t4",
+		},
+		TableKeys: map[string]string{
+			"t1": "k1", "t4": "k4",
+		},
+	}.Validate()
+
+	errs := err.(interface{ Unwrap() []error }).Unwrap()
+
+	is.True(len(errs) == 2)
+	is.Equal(errs[0].Error(), `missing key for table "t2"`)
+	is.Equal(errs[1].Error(), `missing key for table "t3"`)
```

Comment:
Nitpick: I _think_ this should work too and simplifies the code a bit:
```go
	is.Equal(
		err,
		errors.Join(errors.New(`missing key for table "t2"`), errors.New(`missing key for table "t3"`)),
	)
```

Reply:
Yea that is easier, thanks!

Reply:
Fixed here f28285404c22064f4ac63af063fcc785358965c3